docs/book/src/topics/managedcluster.md

```diff
@@ -201,6 +203,41 @@ should be fairly clear from context.
 | networkPlugin             | azure, kubenet                |
 | networkPolicy             | azure, calico                 |
 
+### AKS Addons
+
+AKS addons are enabled via the [addonProfiles](https://learn.microsoft.com/cli/azure/aks/addon?view=azure-cli-latest#az-aks-addon-list-available) specification. For more information about required config values and addons which may not be listed below, look for the `*ADDON_NAME` values in [this code](https://github.com/Azure/azure-cli/blob/main/src/azure-cli/azure/cli/command_modules/acs/_consts.py).
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureManagedControlPlane
+metadata:
+  name: my-cluster-control-plane
+spec:
+  location: southcentralus
+  resourceGroupName: foo-bar
+  subscriptionID: 00000000-0000-0000-0000-000000000000 # fake uuid
+  version: v1.21.2
```

Comment:
this version of k8s is pretty old, I know it's only an example but maybe change it to something more recent or a fake version